src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -265,12 +256,23 @@
             consumes = APPLICATION_JSON_UTF8_VALUE,
             produces = APPLICATION_JSON_UTF8_VALUE
     )
+
     @ResponseBody
-    public ResponseEntity<CreateUserProfileResponse> updateUserProfile(@Valid @RequestBody UpdateUserProfileData updateUserProfileData, @PathVariable String userId) {
+    public ResponseEntity<UserProfileRolesResponse> updateUserProfile(@Valid @RequestBody UpdateUserProfileData updateUserProfileData, @PathVariable String userId
+    ) {
         log.info("Updating user profile");
+        UserProfileRolesResponse userProfileResponse = new UserProfileRolesResponse();
+        if (StringUtils.isEmpty(updateUserProfileData.getRolesAdd())
```

Comment:
I am not sure but does StringUtils also check elements in Set as empty ? Can we use CollectionUtils.isEmpty ?

Reply:
removed the StringUtils and added collectionUtils.isEmpty check